controllers/opportunity/index.js

```diff
@@ -64,6 +64,49 @@ class opportunityController {
       });
     }
   }
+
+  //updateOpportunities
+async updateOpportunity(req,res){
+
+  let opportunityId=req.params.opportunityId;
+  let queryObject={opportunityId:opportunityId}
```

Comment:
Instead of using opportunityId for updating, use the default id generated by mongo (`_id`)